omnigibson/__init__.py

```diff
@@ -52,8 +53,8 @@ def cleanup(*args, **kwargs):
         shutil.rmtree(tempdir)
     except PermissionError:
         log.info("Permission error when removing temp files. Ignoring")
-    from omnigibson.utils.ui_utils import suppress_omni_log
-    log.info(f"{'-' * 10} Shutting Down OmniGibson {'-' * 10}")
+    from omnigibson.simulator import logo_small
+    log.info(f"{'-' * 10} Shutting Down {logo_small()} {'-' * 10}")
```

Comment:
I actually changed this explicitly because of an issue with circular imports - also feels weird for the simulator to need to be imported here now. (also the "as launch" part is unused). Could you put the logos in a separate utils file somewhere actually?

Reply:
done